spring-boot-modules/spring-boot-springdoc/pom.xml

```diff
@@ -119,53 +118,6 @@
         </resources>
     </build>
 
-    <profiles>
-        <profile>
-            <id>integration</id>
-            <build>
```

Comment:
Why has this profile been eliminated?

Reply:
I migrated it to the other module.

Reply:
My question is: isn't any other article/code taking advantage of that profile?

Reply:
No, the integration with the plugins is part of this article. Other articles don't refer to this part.